apps/web/pages/event-types/index.tsx

```diff
@@ -55,293 +36,6 @@ const CreateFirstEventTypeView = ({ canAddEvents, profiles }: CreateEventTypePro
   );
 };
 
-type EventTypeGroup = inferQueryOutput<"viewer.eventTypes">["eventTypeGroups"][number];
-type EventType = EventTypeGroup["eventTypes"][number];
-interface EventTypeListProps {
-  profile: { slug: string | null };
-  readOnly: boolean;
-  types: EventType[];
-}
-const EventTypeList = ({ readOnly, types, profile }: EventTypeListProps): JSX.Element => {
```

Comment:
I'm assuming this got extracted to a an external file. We kinda have an unspoken rule about separating components. We only do it once it is needed in more than one place. If it's not, let's keep it in the same file for now.